src/test/java/com/mongodb/hibernate/jdbc/MongoStatementIntegrationTests.java

```diff
@@ -0,0 +1,349 @@
+            sessionFactory.close();
+        }
+    }
+
+    @BeforeEach
+    void setUp() {
+        session = sessionFactory.openSession();
+    }
+
+    @AfterEach
+    void tearDown() {
+        if (session != null) {
+            session.close();
+        }
+    }
+
+    private static final String COLLECTION_NAME = "books";
+
+    @Nested
+    class ExecuteUpdateTests {
+
+        @BeforeEach
+        void setUp() {
+            session.doWork(conn -> {
+                conn.createStatement()
+                        .executeUpdate(
+                                """
+                        {
+                            delete: "%s",
+                            deletes: [
+                                { q: {}, limit: 0 }
+                            ]
+                        }"""
+                                        .formatted(COLLECTION_NAME));
+            });
+        }
+
+        private static final String INSERT_MQL =
+                """
+                 {
+                            insert: "%s",
+                            documents: [
+                                {
+                                    _id: 1,
+                                    title: "War and Peace",
+                                    author: "Leo Tolstoy",
+                                    outOfStock: false
+                                },
+                                {
+                                    _id: 2,
+                                    title: "Anna Karenina",
+                                    author: "Leo Tolstoy",
+                                    outOfStock: false
+                                },
+                                {
+                                    _id: 3,
+                                    title: "Resurrection",
+                                    author: "Leo Tolstoy",
+                                    outOfStock: false
+                                },
+                                {
+                                    _id: 4,
+                                    title: "Crime and Punishment",
+                                    author: "Fyodor Dostoevsky",
+                                    outOfStock: false
+                                },
+                                {
+                                    _id: 5,
+                                    title: "The Brothers Karamazov",
+                                    author: "Fyodor Dostoevsky",
+                                    outOfStock: false
+                                },
+                                {
+                                    _id: 6,
+                                    title: "Fathers and Sons",
+                                    author: "Ivan Turgenev",
+                                    outOfStock: false
+                                }
+                            ]
+                        }"""
+                        .formatted(COLLECTION_NAME);
```

Comment:
btw, you can see `Palantir` formatting gradle plugin handles the Java docstring perfectly. It shines in complicated lambda formatting scenario though.